views/app.jade

```diff
@@ -1,15 +0,0 @@
-doctype html
-html(manifest=o.htmlWebpackPlugin.files.manifest)
-  head
-    meta(charset="utf-8")
-    title= o.htmlWebpackPlugin.options.title || 'Simplenote'
-    //- iOS webviews do not support indexedDB!
-    //- meta(name="apple-mobile-web-app-capable" content="yes")
```

Comment:
sweet. let's just remove this then; though a comment here in the PR/commit would probably be valuable